packages/app-content-pages/src/screens/About/components/HowItWorks.js

```diff
@@ -0,0 +1,310 @@
+            margin='0'
+            pad='0'
+            height='100%'
+          >
+            <ArrowBox>
+              <ArrowSVG />
+              <Text color='white' weight='bold' size='1.125rem'>
+                {`${t('AboutPage.howItWorks.participants.description')}:`}
+              </Text>
+            </ArrowBox>
+            <ListItem>
+              <Compass color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.participants.steps.one')}</Step>
+            </ListItem>
+            <ListItem>
+              <Info color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.participants.steps.two')}</Step>
+            </ListItem>
+            <ListItem>
+              <Compliance color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.participants.steps.three')}</Step>
+            </ListItem>
+            <ListItem>
+              <Certificate color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.participants.steps.four')}</Step>
+            </ListItem>
+          </Box>
+
+          {/** For Researchers */}
+          <Box
+            as='ul'
+            width={{ max: '240px' }}
+            margin='0'
+            pad='0'
+            height='100%'
+          >
+            <ArrowBox>
+              <ArrowSVG />
+              <Text color='white' weight='bold' size='1.125rem'>
+                {`${t('AboutPage.howItWorks.researchers.description')}:`}
+              </Text>
+            </ArrowBox>
+            <ListItem>
+              <Troubleshoot color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.researchers.steps.one')}</Step>
+            </ListItem>
+            <ListItem>
+              <Validate color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.researchers.steps.two')}</Step>
+            </ListItem>
+            <ListItem>
+              <Deploy color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.researchers.steps.three')}</Step>
+            </ListItem>
+            <ListItem>
+              <Download color='white' size='50px' />
+              <Step>{t('AboutPage.howItWorks.researchers.steps.four')}</Step>
+            </ListItem>
+          </Box>
+        </ListsContainer>
+
+        {/** Buttons */}
+        <Box
+          direction='row'
+          pad={{ vertical: 'medium' }}
+          gap='small'
+          justify='between'
+        >
+          <StyledButton
+            as={Anchor}
+            color='black'
+            plain
+            href='https://www.zooniverse.org/projects'
+            label={t('AboutPage.howItWorks.participants.link')}
+            weight='normal'
+          />
+          <StyledButton
+            as={Anchor}
+            color='black'
+            plain
+            href='https://www.zooniverse.org/lab'
+            label={t('AboutPage.howItWorks.researchers.link')}
+            weight='normal'
+          />
```

Comment:
According to the Figma file this link should be teal instead of orange.. Not sure if this was talked about though so I'm simply making a note.

Reply:
Right! I skipped over this because it's difficult to give a `styled` grommet button a custom border when `as={Anchor}`, but I found a way to do so 👍